src/debugAdapter/debugNetwork.ts

```diff
@@ -16,10 +23,10 @@ export class DebugNetwork {
     this.workingDirectory = truffleConfigDirectory;
   }
 
-  public async load(): Promise<void> {
+  public async load(providerUrl?: string): Promise<void> {
```

Comment:
Hey @acuarica,

Yes, we need. This function has been called twice. If you get the branch and debug, you can see there is one routine to set the debugger and another to attach. To set the debugger is not necessary inform this parameter, but In the attach routine, you need to provide the url. It's a redundancy and there is a comment telling to refectory this, but for now, it's impossible to rewrite all the debugger.

But, yes, we must have this parameter here.

Reply:
Yes, I agree. We can have Helper to provide this information. We need to rethink that because there are a bunch of places calling the same information but in different ways...